### ISSUE TITLE
Homekit[IP] : discovery error on windows 

### ISSUE THREAD

**Issue by belkinhappyzj** [jlusiardi/homekit_python#195]
Hi, my first time to use homekit_python, managed to install everything on windows platform.
Tried a discovery test "python -m homekit.discover" but got folowing error:

_Traceback (most recent call last):
  File "C:\Users\STEVENZHAO\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\STEVENZHAO\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\STEVENZHAO\AppData\Roaming\Python\Python38\site-packages\homekit\discover.py", line 51, in <module>
    results = Controller.discover(args.timeout)
  File "C:\Users\STEVENZHAO\AppData\Roaming\Python\Python38\site-packages\homekit\controller\controller.py", line 91, in discover
    return discover_homekit_devices(max_seconds)
  File "C:\Users\STEVENZHAO\AppData\Roaming\Python\Python38\site-packages\homekit\zeroconf_impl\__init__.py", line 102, in discover_homekit_devices
    data.update(parse_discovery_properties(decode_discovery_properties(
  File "C:\Users\STEVENZHAO\AppData\Roaming\Python\Python38\site-packages\homekit\zeroconf_impl\__init__.py", line 181, in parse_discovery_properties
    data['category'] = Categories[int(category)]
  File "C:\Users\STEVENZHAO\AppData\Roaming\Python\Python38\site-packages\homekit\model\categories.py", line 104, in __getitem__
    raise KeyError('Item {item} not found'.format(item=item))
KeyError: 'Item 33 not found'_

**Comment by jlusiardi** [comment]
Hi,

seems that Apple introduced a new Category again. HomeKit Accessory Protocol Specification Non-Commercial Version Release R2 says, categories 33 and above are reserved ;) 

Can you tell me what kind of accessory in your WiFi might be the one to return category 33?

**Comment by belkinhappyzj** [comment]
Hi jlusiardi ,most likely, it is the Eero mesh router in my local network, their latest FW has put in support of homekit: https://support.eero.com/hc/en-us/articles/360036319531-How-to-use-HomeKit-with-your-eero-Network, I guess the category 33 is "Wi-Fi Router", can we update homekit_python to support it?  thank you.

**Comment by jlusiardi** [comment]
I will do this.

**Comment by belkinhappyzj** [comment]
Hi jlusiardi, one quick quesiton regarding TLV8 value when writing charactistic, I am a newbie in HAP protocol,  can you show me some example code for writing certain charactistic using TLV8 value?  Is the formed TLV8 bytes needs to be further encoded?  thank you.

**Comment by jlusiardi** [comment]
Can you please try the code from master if that works?

Perhaps you can have a look at https://github.com/jlusiardi/homekit_python/blob/camera_client/homekit/get_camera_stream.py and the other code from the camera_client branch. I need to write TLV8 code stuff to some characteristics there.

**Comment by jlusiardi** [comment]
how was your experience with the updated code. Does it discover? Does pairing work?

**Comment by belkinhappyzj** [comment]
Hi jlusiardi , I see you added category 33 in categories.py, now the discovery works.  However pairing doesn't work.  I always got a HTTP 500 return on pairing M4 procedure.  I was able to pair it using HAT tool, after comparing the logs, I find HAT tool is using kTLVType_Method = 0 in M1 while Homekit_python use kTLVType_Method = 1, not sure if this is the root cause.  I created another post asking how to try out kTLVType_Method = 0 in M1 with Homekit_lib :
https://github.com/jlusiardi/homekit_python/issues/199

Thank you
